components/UI/CopyToClipboard.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Clipboard from '../../assets/icons/clipboard.svg';
+import TextButton from './TextButton';
+
+const CopyToClipboard = ({ content, title }) => {
+    const [isContentCopied, setIsContentCopied] = useState(false);
+    const { t } = useTranslation();
+
+    const copyToClipboard = async () => {
+        navigator.clipboard.writeText(content);
+        setIsContentCopied(true);
+        await new Promise(r => setTimeout(r, 2000));
+        setIsContentCopied(false);
+    };
+
+    return (
+        <TextButton title={title || t('Copy link to clipboard')} onClick={copyToClipboard}>
```

Comment:
If I see this correctly then we're missing a translation for "Copy link to clipboard"... that should probably go to `_defaults.json`